Fakebook.Profile/Fakebook.Profile.DataAccess/Fakebook.Profile.DataAccess.csproj

```diff
@@ -1,7 +1,13 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.7.0" />
+    <PackageReference Include="MongoDB.Bson" Version="2.12.0-beta1" />
+    <PackageReference Include="MongoDB.Driver" Version="2.12.0-beta1" />
```

Comment:
you've chosen mongodb for the profile service over sql after consideration, or without consideration?